charts/beyla/values.yaml

```diff
@@ -59,13 +59,22 @@ podSecurityContext: {}
   # fsGroup: 2000
 
 securityContext:
+  # -- For an unprivileged / less privileged setup, use privileged: false and uncomment the required capabilities in the values file.
```

Comment:
Could we instead of make users uncomment this add them conditionally?

Reply:
That is absolutely possible. In that case I would propose the following:

We don't use the condition based on `securityContext.privileged` anymore, instead we add a top-level attribute called `privileged` and set it by default true which results in the setup like before. If this is set to `false`, we set the securityContext how it needs to be.

In order to make adding the capabilities `SYS_RESOURCE` and `SYS_ADMIN` optional, because they are not always required, we can add a property `extraCapabilities` that allows for adding those two if needed and other ones.

The only 'downside' I see from this approach is that the control over securityContext is removed from the user once the new `privileged` attribute is set to false. But I would almost say that is okay, seeing as an unprivileged setup requires a very specific set of configurations to work.

Reply:
Sounds reasonoble to me @marevers ! thanks